policies/github/organization.rego

```diff
@@ -110,3 +110,36 @@ default organization_not_using_single_sign_on := true
 organization_not_using_single_sign_on := false {
 	input.saml_enabled
 }
+
+# METADATA
+# scope: rule
+# title: Organization Has Stale Secrets
+# description: Some of the organizations secrets have not been updated for over a year. It is recommended to refresh secret values regularly in order to minimize the risk of breach in case of an information leak.
+# custom:
+#   requiredEnrichers: [secretsList]
+#   remediationSteps:
+#      - Enter your organization's landing page
+#      - Go to the settings tab
+#      - Under the 'Security' title on the left, choose 'Secrets and variables'
+#      - Click 'Actions'
+#      - Sort secrets by 'Last Updated'
+#      - Regenerate every secret older than one year and add the new value to GitHub's secret manager
+#   severity: MEDIUM
+#   requiredScopes: [repo]
+#   threat: Sensitive data may have been inadvertently made public in the past, and an attacker who holds this data may gain access to your current CI and services. In addition, there may be old or unnecessary tokens that have not been inspected and can be used to access sensitive information.
+organization_secret_is_stale[stale] := true{
+    some index
+    secret := input.organization_secrets[index]
+    is_stale(secret.updated_at)
+    stale := {
+    "name" : secret.name,
+    "update date" : time.format(secret.updated_at),
+    }
+
+}
+
+is_stale(date) {
```

Comment:
move to common and use simpler approach to date comparison as we do in the collaborator policy